lib/WTSI/NPG/HTS/TreePublisher.pm

```diff
@@ -171,6 +209,10 @@ has 'require_checksum_cache' =>
       $num_errors    += $ne;
     }
 
+    if (defined $self->mlwh_json) {
```

Comment:
This should test the attribute's predicate.

Reply:
Replaced with predicate check